koku/sources/api/serializers.py

```diff
@@ -80,7 +80,8 @@ class AdminSourcesSerializer(SourcesSerializer):
 
     def validate_source_type(self, source_type):
         """Validate credentials field."""
-        if source_type.lower() in LCASE_PROVIDER_CHOICE_LIST:
+        # Remove OCI from here after db migration
+        if source_type.lower() in LCASE_PROVIDER_CHOICE_LIST and source_type.lower() != "oci":
```

Comment:
Given that this is the admin sources serializer, this is only available for local dev. I'd be fine with us leaving this alone.

Reply:
Sure we can do that. Thanks for the review Michael :bow: